src/Mod/CAM/App/Area.cpp

```diff
@@ -81,13 +78,6 @@
 #include "Area.h"
 
 
-// FIXME: ISO C++11 requires at least one argument for the "..." in a variadic macro
-#if defined(__clang__)
```

Comment:
This doesn't mean that it works without raising a warning with C++17. When compiling the sources with clang and the option `-Werror` this causes the build to fail.